content/articles/concept-of-first-hop-redundancy-protocol/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: concept-of-first-hop-redundancy-protocol/
+title: Concept of First Hop Redundancy Protocol
+description: Organizations use various protocols to prevent a single point failure in their systems. One of these protocols is the First Hop redundancy Protocol (FHRP) which presents a virtual default gateway to the organization's network to provide a near 100 percent network uptime. In this article we discuss FHRP in detail and how it is used to achieve redundancy in networks.
+author: rabo-james-bature
+date: 2021-06-10T00:00:00-18:00
+topics: [Networking]
+excerpt_separator: <!--more-->
+images:
+
+   url: hero.jpg
+   alt: First Hop Redundancy Protocol example image
+
+--- 
+
+Organizations use various protocols to prevent a single point failure in their works. One of these protocols is the First Hop redundancy Protocol (FHRP) which presents a virtual default gateway to the organization's network to provide a near 100 percent network uptime.
+
+This article will look at how FHRP is used to achieve redundancy in networks, variety of FHRP, and then configure Hot Standby Redundancy Protocol (HSRP), a Cisco proprietary protocol used in the implementation of FHRP.
+
+### Table of contents
+
+- [Goals](#goals)
+- [Prerequisite](#prerequisite)
+- [Requirements](#requirements)
+- [Default gateway](#default-gateway)
+- [First Hop Redundancy Protocol (FHRP)](#first-hop-redundancy-protocol-fhrp)
+- [Various First Hop Redundancy Protocols](#various-first-hop-redundancy-protocols)
+  - [Host Standby Router Protocol (HSRP)](#host-standby-router-protocol-hsrp)
+  - [Gateway Load Balancing Protocol (GLBP)](#gateway-load-balancing-protocol-glbp)
+  - [ICMP Router Discovery Protocol (IRDP)](#icmp-router-discovery-protocol-irdp)
+- [HSRP features](#hsrp-features)
+  - [HSRP preemption](#hsrp-preemption)
+- [Steps to configuring HSRP](#steps-to-configuring-hsrp)
+- [Configuring HSRP protocol on a Cisco router](#configuring-hsrp-protocol-on-a-cisco-router)
+- [Conclusion](#conclusion)
+- [Activity File](#activity-file)
+- [Further reading](#further-reading)
+
+### Goals
+At the end of this article, the reader is expected to learn:
+- What FHRP is
+- Various FHRP options available
+- How to configure HSRP on a Cisco router.
+  
+### Prerequisite
```

Comment:
Also, guide the user to set up the system accordingly to proceed with the tutorial.

Reply:
@lalith1403 part of the prerequisite for this tutorial is "basic understanding of the Cisco command line". Anyone with this knowledge will be able to understand this tutorial because a pre-configured  file-link was made available together with the software to be used